test/StripTagsTest.php

```diff
@@ -511,6 +511,25 @@ public function testMultiQuoteInput()
         $this->assertEquals($expected, $filter->filter($input));
     }
 
+    public function testEmptyCommentTags()
+    {
+        $input    = 'Bad <!--> comment';
```

Comment:
Please use dataProvider instead, it will be like that:

```php
    public function comment()
    {
        return [
            ['Bad <!--> comment', 'Bad  comment'],
            ['Bad <!---> comment', 'Bad  comment'],
            ['Bad <!----> comment', 'Bad  comment'],
            ['Bad <!-- --> comment', 'Bad  comment'],
        ];
    }

    /**
     * @dataProvider comment
     *
     * @param string $input
     * @param string $expected
     */
    public function testFilter($input, $expected)
    {
        $this->assertEquals($expected, $this->_filter->filter($input));
    }
```

Reply:
thanks, done.